dynamic_programming/longest_palindromic_substring.cpp

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
follow header formats!
https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md
check the contribution guidelines or other code in this repository for more info